artemis-docker/readme.md

```diff
@@ -1,140 +1,62 @@
 # Docker Image Example
 
 This is an example on how you could create your own Docker Image For Apache
-ActiveMQ Artemis based on CentOS or Ubuntu (Eclipse Temurin JDK images).
+ActiveMQ Artemis based on Alpine, CentOS or Ubuntu (Eclipse Temurin JDK images).
 
-# Preparing
+# Selecting the image
 
-Use the script ./prepare-docker.sh as it will copy the docker files under the
-binary distribution.
+This folder contains four different images for hosting your ActiveMQ Artemis instance in docker
 
-Below is shown the command to prepare the build of the Docker Image starting
-from the local distribution (from the source codes of ActiveMQ Artemis)
+- Alpine
+- CentOS7
+- Ubuntu 11 with JDK
+- Ubuntu 11 with JRE
```

Comment:
`11` isn't the version of Ubuntu. It is the version of _Java_. Therefore, the list should probably be:
 - Alpine (latest) with JRE 11
 - CentOS 7 with JDK 11
 - Ubuntu (latest LTS) with JDK 11
 - Ubuntu (latest LTS) with JRE 11

Reply:
You're right.